backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1714,4 +1714,22 @@ databaseChangeLog:
               now() updated_at,
               migrations_user_id updated_by
             FROM ${database.defaultSchemaName}.person CROSS JOIN migration_user
-            WHERE test_result_delivery IS NOT NULL;
\ No newline at end of file
+            WHERE test_result_delivery IS NOT NULL;
+  - changeSet:
+      id: add-org-verified
+      author: jeremy.a.zitomer@omb.eop.gov
+      comment: Add a required boolean column to the organization table named identity_verified, existing records set to true.
+      changes:
+        - addColumn:
+            tableName: organization
+            columns:
+              - column:
+                  name: identity_verified
+                  type: boolean
+                  remarks: Whether this organization's identity has been verified.
+                  valueBoolean: true
+                  # in the next commit, will change this default value to false
+                  defaultValueBoolean: true
+        - addNotNullConstraint:
```

Comment:
I would have thought we could put this into the column definition directly, since there's a default value. Does that not work?

Reply:
it didn't even occur to me we could do it all in one step, even with a default value. turns out we can! thank you.